.github/workflows/container-build-quay-io.yml

```diff
@@ -31,12 +31,18 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
+    - name: Install qemu dependency
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y qemu-user-static
```

Comment:
This installation is emulated though, at least it seems (with the emulated runtime for rpm, but also installation scriptlets, etc.). This could make quite some difference compared to the official Fedora images..?

Reply:
> What difference should it do if we use emulated runtimes? 

I don't have anything specific in mind. It's just a bad feeling :-) because it's quite common for Mock builds with forcearch to fail for various reasons.

For example, the entire DNF in the build container is emulated (including RPM) for additional package installation. DNF/RPM interacts with the underlying databases, and these databases are supposed to be utilized by native executors later on.

I don't know; I wouldn't love it if my overexcitement caused issues with our production Copr builders :-) nb, we should have a "natively built" DNF5-based default Fedora 41 image soon.

Reply:
> I don't know; I wouldn't love it if my overexcitement caused issues with our production Copr builders :-) nb, we should have a "natively built" DNF5-based default Fedora 41 image soon.

Agree, I'll dig some more I guess